eth/gasprice/gasprice.go

```diff
@@ -32,12 +32,14 @@ import (
 const sampleNumber = 3 // Number of transactions sampled in a block
 
 var DefaultMaxPrice = big.NewInt(500 * params.GWei)
+var DefaultIgnorePrice = big.NewInt(2 * params.GWei)
```

Comment:
This would bump the current default (`2`) with nine orders of magnitude. I think it's unintentional? 

For sure, `2` might be too low, but let's change these things slowly. I think getting rid of `0` and `1` already will make a big difference. Let's gather some data before bumping it another time

Reply:
Ah yes, will fix :)

Reply:
Fixed and rebased in 25bd9cd677c70d55dab4b9ab00f2556ef20417a9